### ISSUE TITLE
Separator between pages

### ISSUE THREAD

**Issue by ShahGDev1999** [elementdavv/chaka#11]
Add separator between pages to distinguish page where it ends and where next page start

**Comment by elementdavv** [comment]
No. I remove the gap between pages specifically to make page move smoothly and inconspicuously.

**Comment by ShahGDev1999** [comment]
but that didn't change anything that disturb scrolling
